packages/amplify_datastore_plugin_interface/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.0-next.4
+
+### Fixes
+- fix(api): model helpers use targetNames in schemas with CPK enabled ([#2559](https://github.com/aws-amplify/amplify-flutter/pull/2559))
```

Comment:
nit: Not related to datastore? Dk if this was from script or manually added. Some test files might have been touched by the diff on this change. no harm I suppose